assets/src/edit-story/components/postLock/postLockDialog.js

```diff
@@ -56,9 +56,11 @@ function PostLockDialog({ open, user, dashboardLink, previewLink, onClose }) {
       actions={
         <>
           <Plain href={dashboardLink}>{__('My Stories', 'web-stories')}</Plain>
-          <Plain href={previewLink} target="_blank" rel="noreferrer">
-            {__('Preview', 'web-stories')}
-          </Plain>
+          {previewLink && (
```

Comment:
Is there a case where a story doesn't have a previewLink? 🤔 

I thought we always had one, so I don't see the need for this check. If it's for tests  thhen I'd update the tests :)

Reply:
I was getting an error, when the story has no loaded yet.

Reply:
Would be good to know why `previewLink` is undefined then

Reply:
> Would be good to know why previewLink is undefined then

It is just this component renders before provider has setup the story. I have seen this happen in the browser as well. It is just everything hasn't loaded yet and we get a notice. The story load very quickly and it fixes itself. 

I could return null and not render anything unless this is set. 

I think I found all the place in the test where preview_link are not returned. But I may have missed something...